content/scalers/aws-kinesis.md

```diff
@@ -0,0 +1,99 @@
++++
+fragment = "content"
+weight = 100
+title = "AWS Kinesis Stream"
+background = "light"
++++
+
+Scale applications based on AWS Kinesis Stream.
+
+**Availability:** v1.0+ | **Maintainer:** Community
+
+<!--more-->
+
+### Trigger Specification
+
+This specification describes the `aws-kinesis-stream` trigger that scales based on the shard count of AWS Kinesis Stream.
+
+```yaml
+triggers:
+  - type: aws-sqs-queue
+    metadata:
+      # Required: streamName
+      streamName: myKinesisStream
+      # Required: awsRegion
+      awsRegion: "eu-west-1"
+      # Optional: Default: 2
+      shardCount: "2"
+```
+
+**Parameter list:**
+
+- `streamName` - Kinesis Stream name
+- `shardCount` - The target value that a Kinesis data streams consumer can handle.
+- `awsRegion` - AWS Region for the Kinesis Stream
+
+### Authentication Parameters
+
+You can use `TriggerAuthentication` CRD to configure the authenticate by providing either a role ARN or a set of IAM credentials.
+
+**Role based authentication:**
+
+- `awsRoleArn` - Amazon Resource Names (ARNs) uniquely identify AWS resource
+
+**Credential based authentication:**
+
+- `awsAccessKeyID` - Id of the user
+- `awsSecretAccessKey` - Access key for the user to authenticate with
+
+The user will need access to read data from AWS Cloudwatch.
```

Comment:
Is this still applicable to get the metrics via Cloudwatch or is this a copy/paste mistake?

Reply:
Let me know what you think here @msfuko and should be good to go

Reply:
Originally I removed the whole sentence. to be more precise, I added "The user will need `DescribeStreamSummary` access to read data from AWS Kinesis Streams." here. hope this looks good.

Reply:
Thanks!